.github/workflows/deployment-template.yml

```diff
@@ -61,7 +63,17 @@ jobs:
           GA_TRACKING_ID: ${{ secrets.ga-tracking-id }}
           KIBANASERVER: ${{ secrets.kibanaserver }}
 
-      - name: Step 2 - Replace Tokens for dashboards
+      - name: Step 2 - Replace Token in opensearch.yml
+        if: ${{ (inputs.deploy-env == 'preview') || (inputs.deploy-env == 'searchapps') || (inputs.deploy-env == 'observability') }}
```

Comment:
so you want to deploy this for preview and searchapps as well? I'm assuming that you only need to focus on dev, observablity, if everything works, then move to Prod if you need. right?